platforms/documentation/docs/src/docs/userguide/running-builds/tutorial/part1_gradle_init.adoc

```diff
@@ -103,17 +102,17 @@ When you are done, the directory should look as follows:
 └── ⋮                       // <8>
 ----
 <1> Project-specific cache directory generated by Gradle.
-<2> The version catalog which defines a set of versions for dependencies in a central location.
-<3> Contains the JAR file and configuration of the <<gradle_wrapper.adoc#gradle_wrapper_reference,Gradle Wrapper>>.
+<2> Contains the JAR file and configuration of the <<gradle_wrapper.adoc#gradle_wrapper_reference,Gradle Wrapper>>.
+<3> The version catalog which defines a set of versions for dependencies in a central location.
 <4> macOS and Linux script for executing builds using the <<gradle_wrapper.adoc#gradle_wrapper_reference,Gradle Wrapper>>.
 <5> Windows script for executing builds using the <<gradle_wrapper.adoc#gradle_wrapper_reference,Gradle Wrapper>>.
 <6> The project's <<settings_file_basics.adoc#sec:settings_file_script, settings file>> where the list of subprojects is defined.
 <7> The source code and build configuration for the Java app.
 <8> Some additional Git files may be present such as `.gitignore`.
 =====
-[.multi-language-sample]
+
 =====
-[source, groovy]
+[source, text]
 ----
 .
 ├── .gradle                 // <1>
```

Comment:
Please do the same change for the Groovy DSL example

Reply:
Done.